sourcecode/hub/app/Models/User.php

```diff
@@ -142,4 +144,18 @@ public static function fromSocial(string $provider, SocialiteUser $details): sel
             "{$provider}_id" => $details->getId(),
         ]);
     }
+
+    /**
+     * Accessor for 'theme` attribute
+     * @return Attribute<string, null> Selected theme, or default theme if unset or not a valid theme
+     */
+    protected function theme(): Attribute
```

Comment:
If the 'default' and 'bootstrap light' options are going to become invalid, then I'd rather we just add a migration to remove all invalid choices from the database, instead of adding magic accessors to handle choices that can no longer be set.

Removing the 'default' option is also risky. Anyone who saves their preferences will now have a theme set, even if they didn't choose one. If the site administrator later changes the default, that default will not apply to these users.